doc/advanced-configuration/Services.md

```diff
@@ -31,6 +34,41 @@ Provides additional commands to mongooseimctl script.
     * `vcard`: Adds `get_vcard`, `get_vcard2`, `get_vcard2_multi`, `set_vcard`, `set_vcard2`, `set_vcard2_multi`
     * `gdpr`: Adds `retrieve_personal_data`
 
-### Example configuration
+#### Example configuration
 ` {service_admin_extra, [{submods, [node, accounts, sessions]}]} `
 
+### service_mongoose_system_metrics
+
+MongooseIM system metrics are being gathered to analyse the trends and needs of our users, improve MongooseIM, and know where to focus our efforts.
+See [System Metrics Privacy Policy](System-Metrics-Privacy-Policy.md) for more details.
+
+#### Options
+* `report` (default: disabled) - Explicit acknowledgement that the metrics are gathered and reported. Enabling this option is silencing the notification reminder that metrics are gathered.
+* `no_report` (default: disabled) - When this option is set, System Metrics Service is not started and metrics are not collected. Having this option configured, stops the notification warning that the functionality is not being used.
```

Comment:
```suggestion
* `no_report` (default: disabled) - When this option is set, System Metrics Service is not started and metrics are not collected. Having this option enabled stops the notification warning that the functionality is not being used.
```

Reply:
I was trying to avoid the syntax that might be misunderstood into {no_report, enabled} which is enabling an option instead of having it configured.

Reply:
Hmmm... Unless somebody changed my code,s enought to have `no_report`, no need to make it a tuple with a redundant second value, and it is still a valid [`proplist`](http://erlang.org/doc/man/proplists.html) according to the erlang docs 🤷‍♂

Reply:
It's `no_report` in the code
what I meant is that `Having option enabled` could be understood as `{no_report, enalbe}`